docs/source/kale.predict.rst

```diff
@@ -14,26 +14,34 @@ kale.predict.class\_domain\_nets module
    :undoc-members:
    :show-inheritance:
 
-kale.predict.isonet module
+kale.predict.decode module
 --------------------------
 
-.. automodule:: kale.predict.isonet
+.. automodule:: kale.predict.decode
    :members:
    :undoc-members:
    :show-inheritance:
 
-kale.predict.losses module
+kale.predict.initialize_nn module
```

Comment:
This API, by its name, is hard to understand under kale.predict as a separate module. Reconsider and/or discuss.